flang/docs/OpenMPSupport.md

```diff
@@ -16,9 +16,7 @@ local:
 
 This document outlines the OpenMP API features supported by Flang. It is intended as a general reference.
 For the most accurate information on unimplemented features, rely on the compiler’s TODO or “Not Yet Implemented”
-messages, which are considered authoritative.  With the exception of a few corner cases, Flang
-offers full support for OpenMP 3.1 ([See details here](#openmp-31-openmp-25-openmp-11)).
-Partial support for OpenMP 4.0 is also available and currently under active development.
+messages, which are considered authoritative. Partial support for OpenMP 4.0 is also available and currently under active development.
```

Comment:
```suggestion
messages, which are considered authoritative. Partial support for OpenMP 4.0 is available and currently under active development.
```

Reply:
I realized I redacted the OpenMP 3.1 line too. I have added the following statement now:

`Currently, Flang offers full support for OpenMP 3.1; partial support for OpenMP 4.0 is also available and currently under active development.`

Is this okay?

Reply:
Looks alrite.

A more formal way would be the following:
```
Flang provides complete implementation of the OpenMP 3.1 specification and partial implementation of OpenMP 4.0, with continued development efforts aimed at extending full support for the latter.
```